num2words/lang_PT_BR.py

```diff
@@ -132,7 +72,7 @@ def merge(self, curr, next):
         return (ctext + ntext, cnum * nnum)
 
     def to_cardinal(self, value):
-        result = super(Num2Word_PT_BR, self).to_cardinal(value)
+        result = lang_PT.Num2Word_EU.to_cardinal(self, value)
 
```

Comment:
Is really better to use the `to_cardinal()` implementation on `Num2Word_EU` instead of that the one on `Num2Word_PT`? They are so different the cardinal numbers between `pt` and `pt_BR`?

Reply:
Both of them use the `Num2Word_EU` implementation and then change the output based on a regex.
I kept the original regex on `pt_BR` but the regex on `pt` is a little different.
In order for everything to work i thought it was better to continue to use `Num2Word_EU` for both of them.

Reply:
@Olos is good for me then! I see you still have the PR marked as **Work In Progress**. If you have finished, change the status, please. Also, I will really appreciate if you can check the missed lines on the coverage by the tests and add some tests for this lines, so we increase the coverage to the maximum, https://coveralls.io/builds/19469219/source?filename=num2words%2Flang_PT.py#L134.

Reply:
@erozqba Added new tests for those lines.